### PR TITLE
Encode boolean values as uppercase string in WMS request.

### PR DESCRIPTION
**Pull request by stou**
Handle WMS server that only accepts boolean values as uppercase strings.

### CODE REVIEW

**Review by ibrierley — APPROVED** [review]
Feels ok to me.

**Comment by JaffaKetchup** [comment]
Forgive me if I'm wrong, but couldn't you just pass in the boolean string as uppercase anyway?

**Comment by ibrierley** [comment]
I think currently it would have to be a bool rather than a string, and hence toString would be lowercase if I'm not mistaken (quite possible).

File: lib/src/layer/tile_layer.dart [hunk 1]
```diff
@@ -349,6 +349,9 @@ class WMSTileLayerOptions {
   /// tile transparency flag
   final bool transparent;
```
Comment (JaffaKetchup) [inline lib/src/layer/tile_layer.dart:350]: @ibrierley @stou Ah yes, I see now :)

**Review by JaffaKetchup — APPROVED** [review]
LGTM. Just waiting on maintainers to return or people to be given write access. See #1117.

**Comment by JaffaKetchup** [comment]
@stou Please pull upstream when possible, then I (or another maintainer) can merge.

**Comment by stou** [comment]
@JaffaKetchup I have merged in the latest. Looks like the analysis and tests passes now :-)

**Pull request merged**